### PR TITLE
Added dev instructions.

### PR DESCRIPTION
**Pull request by adriangodong**

### CODE REVIEW

**Comment by adriangodong** [comment]
@chrislim LMK if the instructions need some fleshing out.

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+The server code for the skill is written in C# and runs on .NET Core 1.0. The server code is then deployed to AWS Lambda and ran as a serverless function.
+
+[![Build status](https://ci.appveyor.com/api/projects/status/ueprnsv2ade3l0b7/branch/master?svg=true)](https://ci.appveyor.com/project/adriangodong/ceaselessalexa/branch/master)
+
+### Developer Workflow for Server Code
+
+0. Download and install [.NET Core SDK](https://www.microsoft.com/net/download/core)
+1. Build and test using the following command:
+
+       dotnet restore && dotnet build && dotnet test Ceaseless.Tests/Ceaseless.Tests.csproj
```
Comment (chrislim) [inline README.md:16]: Looking good, nice to see the Test Suite passing :)

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+
+2. To publish code to AWS Lambda, run the following command:
+
+       cd Ceaseless
+       dotnet lambda deploy-function
```
Comment (chrislim) [inline README.md:21]: Made it all the way to this step. Getting this error:
No executable found matching command "dotnet-lambda"

**Review by chrislim — COMMENTED** [review]
I went through the steps and got stuck on the one I commented on.

File: README.md (hunk 2 above)
Comment (adriangodong) [reply README.md:21]: (1) Are you working on this branch? I've just added the tool on this branch.
(2) Make sure you're under the Ceaseless folder, e.g. `CeaselessAlexa/Ceaseless`.

Comment (chrislim) [reply README.md:21]: Ah i switched to instructions and reran and it worked.

**Review by chrislim — APPROVED** [review]

**Comment by adriangodong** [comment]
Awesome. I still owe you the nodejs version.

**Pull request merged**